src/parser.js

```diff
@@ -218,6 +219,31 @@ function runInSandbox({ markup, query, cacheId }) {
   sandbox.ensureMarkup(markup);
 
   const result = sandbox.eval(query);
+  result.markup = markup;
+  result.query = query;
```

Comment:
Do we need these two lines? `sandbox.eval` should already return a result containing the `markup` and `query`.

```suggestion
```